nemo_skills/pipeline/backends/slurm.py

```
@@ -0,0 +1,259 @@
+        - containers: Dict mapping container types to images
+        - mounts: List of volume mounts
+        - env_vars: List of environment variables
+        - timeouts: Dict of partition-specific timeouts
+    """
+
+    def __init__(self, cluster_config: Dict):
+        self.config = cluster_config
+
+        # Validate required fields
+        if cluster_config.get("executor") != "slurm":
+            raise ValueError("SlurmBackend requires executor='slurm' in config")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)